pages/common/godot.md

```diff
@@ -0,0 +1,23 @@
+# godot
+
+> An open source 2D and 3D game engine.
+
+- Run a project if the current directory contains a `project.godot` file, or open the project manager:
+
+`godot`
+
+- Edit a project (the current directory must contain a `project.godot` file):
+
+`godot -e`
+
+- Open the project manager even if the current directory contains a `project.godot` file:
```

Comment:
Why `even if` here?

I'm not familiar with godot, so this might make perfect sense to the average beginner. Just want to check :-)

Reply:
Nevermind about this one - looks like @pxgamer picked up on it above :P